crv/crvAdapt.cc

```diff
@@ -194,6 +195,16 @@ static void flagCleaner(crv::Adapt* a)
   }
 }
 
+void getAllBezierFields(ma::Mesh* m, std::vector<apf::Field*>& fields)
+{
+  for (int i = 0; i < m->countFields(); i++) {
+    apf::FieldShape* fs = apf::getShape(m->getField(i));
+    std::string name = fs->getName();
+    if (name == std::string("Bezier"))
```

Comment:
@avinmoharana 
would this also add the coordinate filed? Is that the intended behavior? If I remember correctly, the coordinate field is handled separately. Can you double-check this?